.github/workflows/signoff-check/signoff-check

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A signoff check
+
+The tool checks if any commit got signoff in a pull request.
+
+NOTE: this script is for github actions only, you should not use it anywhere else.
+"""
+import os
+import re
+import sys
+from argparse import ArgumentParser
+
+from github import Github
+
+parser = ArgumentParser(description="signoff check")
+parser.add_argument("--owner", help="repo owner", default='')
+parser.add_argument("--repo_name", help="repo name", default='')
+parser.add_argument("--token", help="github token, will use GITHUB_TOKEN if empty", default='')
+parser.add_argument("--pull_number", help="pull request number", type=int)
+args = parser.parse_args()
+
+OWNER = args.owner if args.owner else os.environ.get('OWNER')
+assert OWNER, 'env OWNER should not be empty'
+REPO_NAME = args.repo_name if args.repo_name else os.environ.get('REPO_NAME')
+assert REPO_NAME, 'env REPO_NAME should not be empty'
+GITHUB_TOKEN = args.token if args.token else os.environ.get('GITHUB_TOKEN')
+assert GITHUB_TOKEN, 'env GITHUB_TOKEN should not be empty'
+PULL_NUMBER = args.pull_number if args.pull_number else int(os.environ.get('PULL_NUMBER'))
+assert PULL_NUMBER, 'env PULL_NUMBER should not be empty'
```

Comment:
Similar "script acts like it's a module but isn't" comment here.  We should either move this code into the `if __name__ == '__main__':` block so it can be safely imported or remove the `__name__` check so it's clear this isn't a module.  My preference is for the former.  If we want to reuse this code in another action, making it import-safe would simplify that reuse.

Reply:
good catch. forgot to refactor this one :P